cypress/support/step_definitions/validations.js

```diff
@@ -319,19 +319,25 @@ Given(
  * @param {String} app - App type
  * @param {String} state - Expected state to be used for validation
  * @example
- * Then '3rd party app' is in 'foreground' state
+ * Then '3rd party app' will stay in 'foreground' state
+ * Then '3rd party app' will be in 'background' state
  */
-Then('{string} is in {string} state', (app, state) => {
+Then(/'(.+)' will (be|stay) in '(.+)' state/, (app, condition, state) => {
   const appId =
     app === CONSTANTS.THIRD_PARTY_APP
       ? UTILS.getEnvVariable(CONSTANTS.THIRD_PARTY_APP_ID)
       : app === CONSTANTS.FIRST_PARTY_APP
         ? UTILS.getEnvVariable(CONSTANTS.FIRST_PARTY_APPID)
         : app;
+  let isEventsExpected;
+  condition == CONSTANTS.STAY ? (isEventsExpected = false) : (isEventsExpected = true);
```

Comment:
Can be done in single line
`const isEventsExpected = condition == CONSTANTS.STAY ? false : true`

Reply:
updated